docs/research/Orange_Pi_flasher.md

```diff
@@ -80,4 +82,82 @@ Reading old flash chip contents... done.
 Erasing and writing flash chip...
 Warning: Chip content is identical to the requested image.
 Erase/write done.
-```
\ No newline at end of file
+```
+
+> Be sure to use stable power supply. Do no supply OrangePi from PC USB.
+> Its efficiency is not enough for proper operating of Orange Pi.
+> It is strongly recommended to use 5V power supply connected to microUSB
+> socket (a trusted USB charger will make it too). Using uncertain power supply
+> leads to flash writing errors, which may brick your target device.
+
+## Customizing kernel for armbian
+
+Clone the repository first and then run script `compile.sh` (must run as root):
+```
+git clone --depth 1 https://github.com/armbian/build
+cd build
+sudo ./compile.sh
+```
+
+`compile.sh` takes care of everything. Downloads cross compilation toolchain and
+all necessary tools.
+
+>They say it works only with Ubuntu Xenial (16.04), other distros are supposed
+to be not supported
+
+This command will pop up a menu. Many options can be chosen there, but the most
+important options are:
+
+1. Select to build only kernel and uboot packages
+2. Show a kernel configuration menu before compilation
+3. Choose target board (in this case `orangepizero`)
+4. Select the target kernel branch as `dev` for newest development version
+5. Accept to enter export mode
+
+>I followed this approach and I strongly recommend to use it this way.
+
+Now the kernel menuconfig should pop up and the customization process begin.
+After saving config and exiting, the kernel compilation will start.
+
+## Updating kernel on Orange Pi Zero
+
+The kernel compilation results in building debian packages in
```

Comment:
'The kernel compilation results, i.e. debian packages are in' - sounds better IMHO